reana-cluster-dev.yaml

```diff
@@ -0,0 +1,73 @@
+cluster:
+  type: "kubernetes"
+  config: "./development-kubeconfig.yaml"
```

Comment:
Shouldn't this be optional? so far it worked out of the box for `minikube`. We can add it to documentation as  "Connecting to remote cluster" or similar.

Reply:
Yes it is optional and if you don't specify this key `reana-cluster` will get the "old" behaviour where configuration to connect to K8S cluster will be fetched from `$HOME/.kube/config`.

I haven't tested if `python-kubernetes` used in `reana-cluster` will honor KUBECONFIG envvar value, which for `kubectl` can be used to specify where `kubectl` should look for kubeconfig to use. (https://kubernetes.io/docs/concepts/configuration/organize-cluster-access-kubeconfig/)